source/help/messaging/formatting-text.rst

```diff
@@ -217,23 +215,32 @@ Renders as: ``monospace``.
 In-line Images
 --------------
 
-Add in-line images by using an exclamation mark followed by the alt text in square brackets then the URL of the image in round brackets. You can add hover text by placing the text in quotes after the link.
+In-line images are images added within lines of text. From Mattermost Server v5.35 and in Mattermost Cloud, you can control whether all in-line images over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` slash commands. 
+
+To add in-line images to text, use an exclamation mark ``!`` followed by the ``alt text`` in square brackets ``[ ]``, then the ``image URL`` in round brackets ``( )``. You can add hover text after the link by placing the text in quotes ``" "``.
+
+Example:
+
+.. code-block:: none
+
+  ![{alt text}]({URL of image} "{Hover text}")
```

Comment:
0/5, but using the curly brackets in the formatting example may seem confusing to an end user

Reply:
I tend to agree given my own experience learning how to add in-line images.